datarobot_batch_scoring/batch_scoring.py

```diff
@@ -889,47 +886,51 @@ def main(argv=sys.argv[1:]):
     parser.add_argument('-n', '--no', dest='prompt', action='store_false',
                         help="Always answer 'no' for user prompts")
 
-    parsed_args = parser.parse_args()
-    level = logging.DEBUG if parsed_args.verbose else logging.INFO
+    parsed_args = {}
+    conf_file = get_config_file()
+    if conf_file:
+        file_args = parse_config_file(conf_file)
+    parsed_args.update(file_args)
+    pre_parsed_args = {k: v for k, v in parser.parse_args()._get_kwargs() if v is not None}
+    parsed_args.update(pre_parsed_args)
+    level = logging.DEBUG if parsed_args['verbose'] else logging.INFO
     configure_logging(level)
-    printed_args = copy.copy(vars(parsed_args))
+    printed_args = copy.copy(parsed_args)
     printed_args.pop('password')
     root_logger.debug(printed_args)
     root_logger.info('platform: {} {}'.format(sys.platform, sys.version))
 
     # parse args
-    host = parsed_args.host
-    pid = parsed_args.project_id
-    lid = parsed_args.model_id
-    n_retry = int(parsed_args.n_retry)
-    if parsed_args.keep_cols:
-        keep_cols = [s.strip() for s in parsed_args.keep_cols.split(',')]
+    host = parsed_args['host']
+    pid = parsed_args['project_id']
+    lid = parsed_args['model_id']
+    n_retry = int(parsed_args['n_retry'])
+    if parsed_args.get('keep_cols'):
+        keep_cols = [s.strip() for s in parsed_args['keep_cols'].split(',')]
     else:
         keep_cols = None
-    concurrent = int(parsed_args.n_concurrent)
-    dataset = parsed_args.dataset
-    n_samples = int(parsed_args.n_samples)
-    delimiter = parsed_args.delimiter
-    resume = parsed_args.resume
-    out_file = parsed_args.out
-    datarobot_key = parsed_args.datarobot_key
-    pwd = parsed_args.password
-    timeout = int(parsed_args.timeout)
-
-    ui = UI(parsed_args.prompt)
-
-    if not hasattr(parsed_args, 'user'):
+
+    concurrent = int(parsed_args['n_concurrent'])
+    dataset = parsed_args['dataset']
+    n_samples = int(parsed_args['n_samples'])
+    delimiter = parsed_args.get('delimiter')
+    resume = parsed_args['resume']
+    out_file = parsed_args['out']
+    datarobot_key = parsed_args.get('datarobot_key')
+    pwd = parsed_args['password']
+    timeout = int(parsed_args['timeout'])
```

Comment:
I believe just specifying `type` in `add_argument()` does all work.

But I ok with making the change in separate PR.